tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_InferredAssetSQLExample.ipynb

```diff
@@ -0,0 +1,1215 @@
+            ]
+          },
+          "execution_count": 19,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "validator.head()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "8be66602",
+      "metadata": {},
+      "source": [
+        "### Typical Workflow\n",
+        "A `batch_list` becomes really useful when you are calculating parameters for auto-initializing Expectations, as they us a `RuleBasedProfiler` under-the-hood to calculate parameters."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "fdef9ad7",
+      "metadata": {},
+      "source": [
+        "Here is an example running `expect_column_median_to_be_between()` by \"guessing\" at the `min_value` and `max_value`. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 20,
+      "id": "b524a2df",
+      "metadata": {},
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "321a3263c34e45baa82808745d99f000",
+              "version_major": 2,
+              "version_minor": 0
+            },
+            "text/plain": [
+              "Calculating Metrics:   0%|          | 0/9 [00:00<?, ?it/s]"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/plain": [
+              "{\n",
+              "  \"success\": false,\n",
+              "  \"exception_info\": {\n",
+              "    \"raised_exception\": false,\n",
+              "    \"exception_traceback\": null,\n",
+              "    \"exception_message\": null\n",
+              "  },\n",
+              "  \"meta\": {},\n",
+              "  \"result\": {\n",
+              "    \"observed_value\": 1.6\n",
+              "  }\n",
+              "}"
+            ]
+          },
+          "execution_count": 20,
+          "metadata": {},
+          "output_type": "execute_result"
+        }
+      ],
+      "source": [
+        "validator.expect_column_median_to_be_between(column=\"trip_distance\", min_value=0, max_value=1)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "f18d096f",
+      "metadata": {},
+      "source": [
+        "The observed value for our `yellow_tripdata_sample_2020_01` table where `vendor_id` = `2`  is going to be `1.6`, which means the Expectation fails"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "9c5016d8",
+      "metadata": {},
+      "source": [
+        "Now we run the same expectation again, but this time with `auto=True`. This means the `median` values are going to calculated across the `batch_list` associated with the `Validator` (ie 3 Batches for `yellow_tripdata_sample_2020_01`), which gives the min value of `1.5` and the max value of `5.23`"
```

Comment:
Is this across three batches or the full 12? Also why is there a _01 suffix? Also small nit:
```suggestion
        "Now we run the same expectation again, but this time with `auto=True`. This means the `median` values are going to be calculated across the `batch_list` associated with the `Validator` (ie 3 Batches for `yellow_tripdata_sample_2020_01`), which gives a min value of the median for any batch of `1.5` and a max value of `5.23`"
```